examples/reagent/README.md

```diff
@@ -0,0 +1,45 @@
+# Reagent TodoMVC Example
```

Comment:
Could you rename this file to `readme.md` (lowercase)? To be consistent with other apps.